docs/_data/table/user-configuration.csv

```diff
@@ -31,7 +31,7 @@ alluxio.user.ufs.delegation.write.buffer.size.bytes,2MB
 alluxio.user.ufs.file.reader.class,alluxio.client.netty.&#8203;NettyUnderFileSystemFileReader
 alluxio.user.ufs.file.writer.class,alluxio.client.netty.&#8203;NettyUnderFileSystemFileWriter
 alluxio.user.rpc.retry.base.sleep.ms,50
-alluxio.user.rpc.retry.max.sleep.ms,5000
+alluxio.user.rpc.retry.max.sleep.ms,30000
```

Comment:
I think it's better to leave them in ms for now to avoid the weirdness of `sleep.ms,30sec`.